OpenSim/Simulation/SimbodyEngine/Joint.h

```diff
@@ -112,6 +112,11 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Joint, ModelComponent);
         "to the Joint. When the joint is delete so are the Frames in this list.");
 
 //=============================================================================
+// OUTPUTS
+//=============================================================================
+    OpenSim_DECLARE_OUTPUT(power, double, calcPower, SimTK::Stage::Acceleration);
```

Comment:
Comment says acceleration because it needs to compute constraint forces, which well depend on applied forces. The constraint multipliers are solved concurrently witch acceleration so I think this is correct.